include/surface_perception/segmentation.h

```diff
@@ -130,13 +130,15 @@ class Segmentation {
 /// \param[in] indices The indices in the point cloud to find a surface in.
 /// \param[in] horizontal_tolerance_degrees The tolerance, in degrees, for a
 ///   surface to be considered horizontal.
-/// \param[out] surface The detected surface (may be changed even if no surface
+/// \param[out] surfaces The vector of detected surfaces (may be changed even if
+/// no surface
 ///   is found).
 ///
 /// \returns true if a surface was found, false otherwise.
 bool FindSurface(pcl::PointCloud<pcl::PointXYZRGB>::Ptr cloud,
```

Comment:
Rename to FindSurfaces. Also, update the documentation above to say "Finds horizontal surfaces in the given point cloud."

Reply:
Completed